spec/models/user_spec.rb

```diff
@@ -17,5 +17,14 @@
     let(:user) { User.create_by_admin(email: 'lola@o2.pl')}
     it { user.valid_password?('secret').eql? true }
   end
+
+  describe '#to_csv' do
```

Comment:
This particular test has no value, you are just testing class of the output. 

You took a very nice approach to test service called by this method, so you already know that output is correct.

Please tell me what is this method doing in one sentence.